README.rst

```diff
@@ -10,24 +10,29 @@ genty
 .. image:: https://pypip.in/d/genty/badge.png
     :target: https://pypi.python.org/pypi/genty
 
-Upcoming Breaking Change!
+Hot in 1.2!
 -------------------------
 
-When Genty was released through version 0.2.0, it was released under the namespace
-box.test. In version 0.3.0, importing genty became easier:
+Parametrized tests where the final parameters are not determined until test
+execution time. It looks like so:
 
 .. code-block:: python
 
-    from genty import genty, genty_dataset, genty_args
-
-vs.
+    @genty_dataset((1000, 100), (100, 1))
+    def calculate(self, x_val, y_val):
+        # when this is called... we've been authenticated
+        return self._some_function(x_val, y_val)
 
-.. code-block:: python
+    @genty_dataprovider(calculate)
+    def test_heavy(self, data1, data2, data3):
+        ...
 
-    from box.test.genty import genty, genty_dataset, genty_args
-    from box.test.genty.genty_args import genty_args
+The ``calculate()`` method is called N times, and each of it's return values
```

Comment:
make this specific to the example - "it's called 2 times, based on the @genty_dataset decorator"